L1Trigger/L1THGCal/src/backend/HGCalStage1TruncationImpl_SA.cc

```diff
@@ -0,0 +1,104 @@
+#include "L1Trigger/L1THGCal/interface/backend/HGCalStage1TruncationImpl_SA.h"
+#include <cmath>
+
+HGCalStage1TruncationImplSA::HGCalStage1TruncationImplSA() {}
+
+unsigned HGCalStage1TruncationImplSA::run(const l1thgcfirmware::HGCalTriggerCellSACollection& tcs_in,
+                                          const l1thgcfirmware::Stage1TruncationConfig theConf,
```

Comment:
Should pass the config by reference to avoid the copy

Reply:
done